automation/src/test/scala/org/broadinstitute/dsde/test/api/MethodLaunchSpec.scala

```diff
@@ -260,12 +260,12 @@ class MethodLaunchSpec
               ignoreEmptyOutputs = false
             )
           )
-          exception.message.parseJson.asJsObject
+          val errmsg = exception.message.parseJson.asJsObject
             .fields("message")
             .convertTo[String]
-            .contains(
-              "The expression in your SubmissionRequest matched only entities of the wrong type. (Expected type sample.)"
-            ) shouldBe true
+
+          errmsg should include
+          "Method configuration expects an entity of type sample, but you gave us participant."
```

Comment:
This (and below) changes the swat test to look for the new wording of the Quicksilver error message, instead of the old wording from legacy. Do you think it would be better to reword Quicksilver's error messages and revert the tests?

Reply:
I think these two error messages seem pretty equivalent, so updating the tests to match is fine